context/app/static/js/hooks/useSortState.ts

```diff
@@ -77,6 +77,7 @@ export const useSortState = (
       : sortState.columnId;
 
   // The doc _id is used as a tiebreaker which is necessary for search_after queries.
+  // TODO: Update types to improve s
```

Comment:
Seems to be incomplete?

Reply:
Yup, I think I got distracted by the portal-containers work halfway through writing this comment 😅